android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/TiListView.java

```diff
@@ -615,6 +627,10 @@ public void processProperties(KrollDict d) {
 			footerView = inflater.inflate(headerFooterId, null);
 			footerView.findViewById(titleId).setVisibility(View.GONE);
 		}
+		
+		if (d.containsKeyAndNotNull(TiC.PROPERTY_CAN_SCROLL)) {
+			canScroll = TiConvert.toBoolean(d.get(TiC.PROPERTY_CAN_SCROLL));
```

Comment:
The default value is already set to true in ListViewProxy line 81: https://github.com/appcelerator/titanium_mobile/pull/8593/files#diff-5bb5193580bd9e5340b4bee58bfd23f4R81 but I'll add it in here too then